modules/chapter2/pages/section3.adoc

```diff
@@ -190,3 +194,4 @@ c. Follow the instructions in the notebook.
 Click the first cell, then press btn:[Shift+Enter] to execute the cell and move to the next one.
 Next, execute and review the rest of the cells.
 Keep pressing btn:[Shift+Enter] until you reach the bottom.
+// Exercise has not end objective or goal that we have achieved.
```

Comment:
Please add appropriate objective to beginning and summarize at the end of the exercise